Ansible/roles/forest_calib/tasks/main.yml

```diff
@@ -43,22 +43,37 @@
     line: '{{ user }} ALL=(ALL) NOPASSWD: ALL'
     validate: 'visudo -cf %s'
 
+- name: Create docker user
+  ansible.builtin.user:
+    name: '{{ user }}'
+    append: true
+    shell: /bin/bash
+    system: true
+    create_home: false
```

Comment:
You previously removed this step. Now you've put it back. Could you explain what it does and why we need it?